pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -139,15 +139,15 @@ func (br *OVSBridge) lookupByName() (bool, Error) {
 	return true, nil
 }
 
-func (br *OVSBridge) updateProtocols() Error {
+func (br *OVSBridge) updateProtocolsAndDatapathType() Error {
 	tx := br.ovsdb.Transaction(openvSwitchSchema)
 	// Use Openflow protocol version 1.0 and 1.3.
 	tx.Update(dbtransaction.Update{
 		Table: "Bridge",
 		Where: [][]interface{}{{"name", "==", br.name}},
 		Row: map[string]interface{}{
 			"protocols": makeOVSDBSetFromList([]string{openflowProtoVersion10,
-				openflowProtoVersion13}),
+				openflowProtoVersion13}), "datapath_type": br.datapathType,
```

Comment:
The format is confusing, I thought `datapath_type` was another argument of `makeOVSDBSetFromList`.
Please move it to a newline aligned with "protocols".

Reply:
Done.

Reply:
Is there an update?

Reply:
Now its updated. Thanks.